src/total.jl

```diff
@@ -44,30 +40,60 @@ function total(x::Symbol, design::SimpleRandomSample)
     end
     m = mean(x,design)
     total = design.popsize * m.mean[1]
-    return DataFrame(total=total, se_total=se_tot(x, design))
+    return DataFrame(total=total, SE=se(x, design))
 end
 
 """
-Inner methods for `by`
-"""
-function se_total_by(x::AbstractVector, design::SimpleRandomSample, _)
-    # vector of length equal to `sampsize` containing `x` and zeros
-    z = cat(zeros(design.sampsize - length(x)), x; dims=1)
-    # variance of the total
-    variance = design.popsize^2 / design.sampsize * design.fpc * var(z)
-    # return the standard error
-    return sqrt(variance)
-end
+Estimate subpopulation total for a stratified sample. 
```

Comment:
This docstring should be general, not only for `StratifiedSample`. We should have the same docstring for the same functionality and give examples showing what types work with the function.

Reply:
Or do it in the next PR.